testkit/src/main/scala/org/bitcoins/testkit/node/NodeUnitTest.scala

```diff
@@ -93,24 +93,24 @@ trait NodeUnitTest extends BaseNodeTest {
     )(test)
   }
 
-  def withSpvNodeConnectedToBitcoindV21(test: OneArgAsyncTest)(implicit
```

Comment:
Why delete the `V21` version? I use this on other projects and don't need to upgrade to `V22` yet